install_scripts/templates/common/kubernetes.sh

```diff
@@ -70,8 +128,16 @@ installCNIPlugins() {
 installKubernetesComponents() {
     case "$LSB_DIST$DIST_VERSION" in
         ubuntu16.04)
+            if commandExists "kubeadm"; then
```

Comment:
Let's move this back inside the function (installComponentsApt)

Reply:
agreed